### PR TITLE
ADFS module - Fix links and https

### PR DESCRIPTION
**Pull request by PatAltimore**
Fixes all ADFS articles:
- [about_CommonParameters] link 
- Changes references to http: to https:

This PR reduces the number of changes in https://github.com/MicrosoftDocs/windows-powershell-docs/pull/2685 to simplify review.

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by PatAltimore** [comment]
Hi @tiburd,

This PR fixes link syntax and changes http links to https. I created this to simplify review of PR https://github.com/MicrosoftDocs/windows-powershell-docs/pull/2685. 

I have one approval. Could you help assign some other reviewers? It should be an easy review and approval.

Thanks,
Pat

**Comment by tiburd** [comment]
@PatAltimore
Hi,
I assigned the 3 technical reviewers.
Johan's review is a copy edit pass.
Thanks!

File: docset/winserver2022-ps/adfs/Add-AdfsLocalClaimsProviderTrust.md
Comment (get-itips) [inline docset/winserver2022-ps/adfs/Add-AdfsLocalClaimsProviderTrust.md:39]: I think that, these, being _Claim Types_, and not _browseable links_, should not be changed to https, more information: 

https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/technical-reference/the-role-of-claims#what-are-claim-types

**Comment by PatAltimore** [comment]
Hi @dariomws, @tseward, @get-itips, @tiburd 

Hold off reviewing. I found out I need to change the http requests back for claims and permit URLs. I'll ping you once I correct it.

Thanks,
Pat

**Comment by tiburd** [comment]
@PatAltimore 
Hi,
I removed the other 2 reviewers and their assignments for now.
Please @ mention me when it's ready to be reviewed, and I'll re-add them.
Thanks!

File: docset/winserver2022-ps/adfs/Add-AdfsLocalClaimsProviderTrust.md
Comment (PatAltimore) [reply docset/winserver2022-ps/adfs/Add-AdfsLocalClaimsProviderTrust.md]: Thanks for catching this. I changed all references back to http://schemas.*.

**Comment by PatAltimore** [comment]
Hi @tiburd, 

I made the fix. You can add the other reviewers. Thanks @get-itips. 

Thanks,
Pat

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Pull request merged**
